src/System.Private.CoreLib/src/System/AppContext/AppContextDefaultValues.Defaults.cs

```diff
@@ -9,6 +9,7 @@ namespace System
     internal static partial class AppContextDefaultValues
     {
         internal static readonly string SwitchNoAsyncCurrentCulture = "Switch.System.Globalization.NoAsyncCurrentCulture";
+        internal static readonly string SwitchEnforceJapaneseEraYearRanges = "Switch.System.Globalization.EnforceJapaneseEraYearRanges";
```

Comment:
Do you think that this has a high probability of being .NET Core 3.0 migration blocker for somebody?

The bar for adding quirks to .NET Core 3.0 is different than for adding quirks to .NET Framework. The default plan for .NET Core is to add quirks re-actively, only once somebody reports it as migration blocker.